interpreter/cling/include/cling/Interpreter/Interpreter.h

```diff
@@ -667,6 +679,9 @@ namespace cling {
                         IgnoreFilesFunc_t ignoreFiles =
                           [](const clang::PresumedLoc&) { return false;}) const;
 
+    void CodeComplete(const std::string& line, size_t& cursor,
```

Comment:
As it's internal I'd prefer it to be private and be called `CodeCompleteInternal`, for symmetry reasons with `DeclareInternal()` etc